File: zeno-habit-api/src/zeno-habit-api-data/Models/Habit.cs
using System;

namespace zeno_habit_api_data.Models
{
    public class Habit : IEntity
    {
        public Guid Id { get; set; }
        public string CreatedByUserId { get; set; }
        public string Name { get; set; }
        public string Question { get; set; }
        public DateTime? CreatedDate { get; set; }
    }
}

File: zeno-habit-web/src/models/Habit.ts
export interface Habit {
    id: string,
    createdByUserId: string,
    name: string,
    question: string,
    createdDate: string,
    archived: boolean
}
export interface HabitCreationProps {
    id: string | undefined,
    name: string,
    question: string
}